hardware/BleBox.cpp

```diff
@@ -168,7 +200,7 @@ std::string BleBox::IPToHex(const std::string &IPAddress)
 	std::vector<std::string> strarray;
 	boost::split(strarray, IPAddress, boost::is_any_of("."));
 	if (strarray.size() != 4)
-		return "0.0.0.0";
+		return "";
```

Comment:
Hmmm then my previous fix was better, but you better handle it.
I also would like to ask if you could use the default'StringSplit' method used in domoticz, instead of the boost one
Also, you input an IPAddress like (1.2.3.4), and then in this function ouput 1.2.3.4 ?
Not sure if i can see a difference...